.env_sample

```diff
@@ -0,0 +1,3 @@
+GOOGLE_ANALYTICS_TRACKING_ID=GA1
```

Comment:
ga's values needs to start with an Integer, I'll make the change in the `docker-compose` default.